src/main/java/club/issizler/okyanus/api/cmdnew/ICommand.java

```diff
@@ -0,0 +1,50 @@
+package club.issizler.okyanus.api.cmdnew;
+
+import java.util.List;
+
+public interface ICommand {
```

Comment:
I never use the `I` prefix when referring to interfaces. You probably shouldn't as well. I feel like it's kind of unneccesary. We don't go around having variables like `publicStringName`, or classes like `FooClass` do we?